### PR TITLE
add ref to pseudocounts

### PR DESCRIPTION
**Pull request by 0xMuluh**
This PR adds context to pseudocount and references: https://www.nature.com/articles/nmeth.2897
ping #566

### CODE REVIEW

File: inst/pages/transformation.qmd [hunk 1]
```diff
@@ -100,14 +100,18 @@ A complete list of available transformations and parameters, is
...
-
-Pseudocount can affect the results. Add here citation to the article. Also
-explain the idea of pseudocount. ancombc2 takes into account the affect by
-performing sensitivity test on multiple pseudocount values.
+::: {.callout-important}
+
+Pseudocount value can have a significant impact on the results when applying 
+a logarithm transformation to normalized data, as the logarithm transformation 
+is a nonlinear operation that can fundamentally change the data distribution. 
+See [Costea et al. 2014](https://www.nature.com/articles/nmeth.2897).
```
Comment (TuomasBorman) [inline inst/pages/transformation.qmd]: Add this to bibliography file and refer to it with [@this_reference]. Check for instance line 68 for reference.

Also add sentence about pseudocount; why it is added

File: inst/pages/transformation.qmd [hunk 2]
```diff
@@ -100,14 +100,18 @@ A complete list of available transformations and parameters, is
...
+Pseudocount value can have a significant impact on the results when applying 
+a logarithm transformation to normalized data, as the logarithm transformation 
+is a nonlinear operation that can fundamentally change the data distribution. 
+See [Costea et al. 2014](https://www.nature.com/articles/nmeth.2897). 
+
+Pseudocount should be chosen consistently across all normalization methods being 
+compared, for example, by setting it to a value smaller than the minimum abundance 
+value before transformation. Some tools, like ancombc2, take into account the effect 
+of the pseudocount by performing sensitivity tests using multiple pseudocount 
+values.
```
Comment (TuomasBorman) [inline inst/pages/transformation.qmd]: refer to differential abundance section

**Review by TuomasBorman — CHANGES_REQUESTED** [review]

File: inst/pages/transformation.qmd (hunk 1 above)
Comment (0xMuluh) [reply inst/pages/transformation.qmd]: Thanks.

File: inst/pages/transformation.qmd (hunk 2 above)
Comment (0xMuluh) [reply inst/pages/transformation.qmd]: ACK.

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**